projects/iio_demo/src.mk

```diff
@@ -27,12 +27,19 @@ SRC_DIRS += $(NO-OS)/network
 SRCS	 += $(NO-OS)/util/circular_buffer.c
 SRCS	 += $(PLATFORM_DRIVERS)/delay.c
 SRCS	 += $(PLATFORM_DRIVERS)/timer.c
+INCS	 += $(INCLUDE)/delay.h			\
```

Comment:
why were these files missing ? was this a hidden problem by the fact that the source files were not linked but all copied ?
if this is the case, then other projects may hide such problems too

Reply:
It was working  because coping src isn't well implemented. I will solve that in a pr for make standalone.